Dangerfile

```diff
@@ -20,14 +20,33 @@ end
 is_adding_draft = !(git.added_files.grep(/_drafts/).empty?)
 is_editing_draft = !(git.modified_files.grep(/_drafts/).empty?)
 if is_editing_draft || is_adding_draft
-  warn("Looks like you are editing a draft. Don't forget to publish when it's finished!")
+    warn("Looks like you are editing a draft. Don't forget to publish when it's finished!")
 end
 
 # -----------------------------------------------------------------------------
 # Check spelling and prose
 # -----------------------------------------------------------------------------
 prose.language = "en-us"
+prose.ignore_numbers = true
```

Comment:
But what if we enter a wrongly-spelled number, Jesse? 🤔

Reply:
I think this is supposed to ignore numerals, not spelled-out numbers. 😄

Reply:
...that was the joke!

Reply:
Clearly not a great joke. It's been a long week, forgive me.